Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -607,10 +568,35 @@ function (FixerInterface $a, FixerInterface $b) {
             }
         }
 
+        $rulesSets = array();
+        foreach (RulesSet::create()->getSetDefinitionNames() as $setName) {
```

Comment:
moving the definition of rule sets to a separate class will probably happen in the future to provide extension point in them to support shared rule sets (for instance a ZF one or a Drupal one). But for now, it indeed looks useless to split it to a separate class (and could even hurt us later by having an api for sets which would not suit the extensible system).